### PR TITLE
PRL-1020:Added changes for Tabs and nextstep dropdown for gatekeeper …

### PR DESCRIPTION
**Pull request by khurshedk**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/PRL-1020

### Change description ###

Added changes for Tabs and nextstep dropdown for gatekeeper and legal advisor

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X ] No
```

### CODE REVIEW

File: definitions/private-law/json/AuthorisationCaseField/ConfidentialTab/AuthorisationCaseField.json [hunk 1]
```diff
@@ -26,5 +26,33 @@
     "CaseFieldID": "childrenConfidentialDetails",
     "UserRole": "caseworker-privatelaw-courtadmin",
     "CRUD": "CRUD"
+  },
+  {
+    "LiveFrom": "02/12/2021",
+    "CaseTypeID": "PRLAPPS",
+    "CaseFieldID": "applicantsConfidentialDetails",
+    "UserRole": "caseworker-privatelaw-judge",
+    "CRUD": "CRUD"
```
Comment (shashisk97) [inline definitions/private-law/json/AuthorisationCaseField/ConfidentialTab/AuthorisationCaseField.json:35]: why do we have CRUD? Readonly is not enough?

File: definitions/private-law/json/AuthorisationCaseField/SummaryTab/AuthorisationCaseField.json [hunk 2]
```diff
@@ -256,7 +256,7 @@
     "CaseTypeID": "PRLAPPS",
     "CaseFieldID": "confidentialDetails",
     "UserRole": "caseworker-privatelaw-courtadmin",
-    "CRUD": "R"
+    "CRUD": "CRUD"
```
Comment (shashisk97) [inline definitions/private-law/json/AuthorisationCaseField/SummaryTab/AuthorisationCaseField.json:259]: same as above comment

File: definitions/private-law/json/CaseTypeTab/PartiesTab/CaseTypeTab.json [hunk 3]
```diff
@@ -18,7 +18,7 @@
     "TabLabel": "Parties",
     "TabDisplayOrder": 11,
     "CaseFieldID": "partiesTabLabel",
-    "TabShowCondition": "caseTypeOfApplication=\"C100\" AND [STATE]!=\"AWAITING_SUBMISSION_TO_HMCTS\" AND [STATE]!=\"AWAITING_RESUBMISSION_TO_HMCTS\"",
+    "TabShowCondition": "caseTypeOfApplication=\"C100\" AND [STATE]=\"SUBMITTED\" OR [STATE]=\"AWAITING_RESUBMISSION_TO_HMCTS\" OR [STATE]=\"CASE_ISSUE\" OR [STATE]=\"CASE_WITHDRAWN\" OR [STATE]=\"SUBMITTED_PAID\" OR [STATE]=\"GATE_KEEPING\"",
```
Comment (shashisk97) [inline definitions/private-law/json/CaseTypeTab/PartiesTab/CaseTypeTab.json:21]: AND and OR combination , will it work?

File: definitions/private-law/json/AuthorisationCaseField/ConfidentialTab/AuthorisationCaseField.json (hunk 1 above)
Comment (khurshedk) [reply definitions/private-law/json/AuthorisationCaseField/ConfidentialTab/AuthorisationCaseField.json:35]: For court admin we kept CRUD, so to keep same access level for gatekeeping staff we placed it.

File: definitions/private-law/json/CaseTypeTab/PartiesTab/CaseTypeTab.json (hunk 3 above)
Comment (khurshedk) [reply definitions/private-law/json/CaseTypeTab/PartiesTab/CaseTypeTab.json:21]: checked in prl environment seems to be working fine..

File: definitions/private-law/json/AuthorisationCaseField/SummaryTab/AuthorisationCaseField.json (hunk 2 above)
Comment (khurshedk) [reply definitions/private-law/json/AuthorisationCaseField/SummaryTab/AuthorisationCaseField.json]: this is not needed here for summary tab so replacing it with "R"

**Pull request merged**